src/System.Private.CoreLib/shared/System/Globalization/CultureData.Unix.cs

```diff
@@ -261,7 +264,7 @@ private static CultureInfo GetUserDefaultCulture()
             return CultureInfo.GetUserDefaultCulture();
         }
 
-        private static string ConvertIcuTimeFormatString(string icuFormatString)
+        private static string ConvertIcuTimeFormatString(ReadOnlySpan<char> icuFormatString)
         {
             StringBuilder sb = StringBuilderCache.Acquire(ICU_ULOC_FULLNAME_CAPACITY);
```

Comment:
Why not stackalloc this as well?

Reply:
Sure.  In this PR I was focused on getting rid of the StringBuilder's used in DllImports.  There are plenty of others in corelib that can be removed otherwise.  But since I'm touching this code anyway, I can get rid of it here.